var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -59,6 +59,11 @@ class NetlibLapack(CMakePackage):
 
     patch('ibm-xl.patch', when='@3.7: %xl')
     patch('ibm-xl.patch', when='@3.7: %xl_r')
+    
+    # https://github.com/Reference-LAPACK/lapack/issues/228
+    # TODO: update 'when' once the version of lapack
```

Comment:
you can update it right now, the version does not have to be in Spack for version comparison to work.

Reply:
I am not sure in which release the fix will appear. It won't necessarily be the next one after 3.8.0 (I don't know their development workflow). In that case, we will patch it too.

Reply:
ok, i just assumed that if it's merged already, it will be there.